specs/eventing/control-plane.md

```diff
@@ -0,0 +1,1120 @@
+   </td>
+  </tr>
+  <tr>
+   <td><code>type</code>
+   </td>
+   <td><code>string</code>
+   </td>
+   <td>The category of the condition, as a short, CamelCase word or phrase.
+<p>
+This is the primary key of the Conditions list when viewed as a map.
+   </td>
+   <td>REQUIRED – No default
+   </td>
+  </tr>
+  <tr>
+   <td><code>status</code>
+   </td>
+   <td>Enum:<ul>
+
+<li>"True"
+<li>"False"
+<li>"Unknown"</li></ul>
+
+   </td>
+   <td>The last measured status of this condition.
+   </td>
+   <td>"Unknown"
+   </td>
+  </tr>
+  <tr>
+   <td><code>reason</code>
+   </td>
+   <td>string
+   </td>
+   <td>One-word CamelCase reason for the condition's last transition.
+   </td>
+   <td>""
+   </td>
+  </tr>
+  <tr>
+   <td><code>message</code>
+   </td>
+   <td>string
+   </td>
+   <td>Human-readable sentence describing the last transition.
+   </td>
+   <td>""
+   </td>
+  </tr>
+  <tr>
+   <td><code>severity</code>
+   </td>
+   <td>Enum:<ul>
+
+<li>""
+<li>"Warning"
+<li>"Info"</li></ul>
+
+   </td>
+   <td>If present, represents the severity of the condition. An empty severity represents a severity level of "Error". 
+   </td>
+   <td>""
+   </td>
+  </tr>
+  <tr>
+   <td><code>lastTransitionTime</code>
+   </td>
+   <td>Timestamp
+   </td>
+   <td>Last update time for this condition.
+   </td>
+   <td>"" – may be unset
+   </td>
+  </tr>
+</table>
+
+Additionally, the resource's `status.conditions` field MUST be managed as
+follows to enable clients (particularly user interfaces) to present useful
+diagnostic and error message to the user. In the following section, conditions
+are referred to by their `type` (aka the string value of the `type` field on the
+Condition).
+
+1.  Each resource MUST have either a `Ready` condition (for ongoing systems) or
+    `Succeeded` condition (for resources that run to completion) with
+    `severity=""`, which MUST use the `True`, `False`, and `Unknown` status
+    values as follows:
+
+    1.  `False` MUST indicate a failure condition.
+    1.  `Unknown` SHOULD indicate that reconciliation is not yet complete and
+        success or failure is not yet determined.
+    1.  `True` SHOULD indicate that the application is fully reconciled and
+        operating correctly.
+
+    `Unknown` and `True` are specified as SHOULD rather than MUST requirements
+    because there may be errors which prevent functioning which cannot be
+    determined by the API stack (e.g. DNS record configuration in certain
+    environments). Implementations are expected to treat these as "MUST" for
+    factors within the control of the implementation.
+
+1.  For non-`Ready` conditions, any conditions with `severity=""` (aka "Error
+    conditions") must be aggregated into the "Ready" condition as follows:
+
+    1.  If the condition is `False`, `Ready` MUST be `False`.
+    1.  If the condition is `Unknown`, `Ready` MUST be `False` or `Unknown`.
+    1.  If the condition is `True`, `Ready` may be any of `True`, `False`, or
```

Comment:
s/may/MAY/  or just remove this line - seems like it doesn't really say much

Reply:
Agreed, but I think spelling out this condition --> Ready for each `status` makes it slightly easier to reason about.